### PR TITLE
Update C# gRPC documentation

### PR DESCRIPTION
**Pull request by dannybabbev**
This PR updates the C# documentation as per the latest standards described in the Microsoft gRPC guide: https://docs.microsoft.com/en-us/aspnet/core/grpc/?view=aspnetcore-3.1

The steps to generate the gRPC C# classes are much simpler than what was in the old documentation and do not require you to manually compile the proto files.

### CODE REVIEW

File: docs/grpc/c#.md [hunk 1]
```diff
@@ -13,67 +13,39 @@ This section enumerates what you need to do to write a client that communicates
...
-Every time you use C# `gRPC`, you will have to import the generated rpc classes, and use `nuget` package manger to install `Grpc.Core` (1.17.0 at time of writing), `Google.Protobuf` (3.6.1), and `Google.Api.CommonProtos` (1.4.0).
-
-```bash
-# from project root, install packages using nuget 
-cd ../
-dotnet add package Grpc.Core --version 1.17.0
-dotnet add package Google.Protobuf --version 3.6.1
-dotnet add package Google.Api.CommonProtos --version 1.4.0
-```
-
```
Comment (guggero) [inline docs/grpc/c#.md:76]: The text below this probably needs some adjusting, because "After installing these" is no longer accurate.

**Review by guggero — APPROVED** [review]
utACK, thanks!

I assume you tested this with an up-to-date setup?
Nice to know that .NET now supports gRPC natively!

File: docs/grpc/c#.md (hunk 1 above)
Comment (dannybabbev) [reply docs/grpc/c#.md:76]: Thanks for the comment. Fixed.

Comment (dannybabbev) [reply docs/grpc/c#.md:76]: Yes, the setup is tested.

File: docs/grpc/c#.md [hunk 2]
```diff
@@ -46,7 +46,7 @@ You're done! Build the project and verify that it works.
 
 #### Imports and Client
 
-After installing these, use the code below to set up a channel and client to connect to your `lnd` node:
+Use the code below to set up a channel and client to connect to your `lnd` node:
```
Comment (bhandras) [inline docs/grpc/c#.md:49]: You could squash this with the prev commit.

**Review by bhandras — APPROVED** [review]
utACK, thanks @dannybabbev

**Comment by bhandras** [comment]
I think you may have a cleaner PR if you'd create a branch on your fork from master, then pull request from there, but it's ok as is too.

**Comment by dannybabbev** [comment]
@bhandras can't you squash all the commits in 1 when merging? Haven't used GitHub this much - I have this future in GitLab :D

**Comment by bhandras** [comment]
> @bhandras can't you squash all the commits in 1 when merging? Haven't used GitHub this much - I have this future in GitLab :D

Yeah, we could do that too, but we usually don't in order to retain a clean commit structure. It's not super important in this case as it's only docs. Thanks again for the PR :+1:

**Pull request merged**